src/lib/pubkey/pk_algs.cpp

```diff
@@ -152,6 +156,17 @@ load_public_key(const AlgorithmIdentifier& alg_id,
       return std::make_unique<XMSS_PublicKey>(key_bits);
 #endif
 
+#if defined(BOTAN_HAS_KYBER)
+   // TODO: those OIDs are taken from thomwiggers/mk-cert for interoperability
+   //       https://github.com/thomwiggers/mk-cert/blob/dd845f3825d8dc5fd69e0e234f6a62266ee4dd4a/algorithms.py#L84
+   if(alg_name == "1.3.6.1.4.1.44363.46.52")
```

Comment:
This should work, I just want to emphasise that these OIDs aren't stable: if I add algorithms to `mk-cert` they tend to move around.

(Also I'm completely unsure whose OID namespace I cribbed)

Reply:
FWIW: For the OIDs in the current Kyber integration in Botan we followed [this IETF draft](https://www.ietf.org/id/draft-uni-qsckeys-00.html#section-4.1).